src/contract/userop/index.ts

```diff
@@ -0,0 +1,188 @@
+// SPDX-License-Identifier: BUSL-1.1
+import { ethers, utils, ContractInterface, Signer } from "ethers";
+import { UserOperationMiddlewareFn, Presets, Client } from "userop";
+import { UserOperationEventEvent } from "userop/dist/typechain/EntryPoint";
+import { VerifiedWallet } from "../../wallet";
+
+enum STATUS {
+    SUCCESS,
+    ERROR
+}
+
+interface SCResponse {
+    response: object;
+    status: STATUS;
+    message: string;
+    reason: string;
+    code: number
+}
+
+enum DATATYPES {
+    VALUE = 'number',
+    STRING = 'string',
+    ADDRESS = 'address',
+}
+
+interface TransactionParams{
+    value:string
+}
+
+const response: { transactionHash: string | null, result: UserOperationEventEvent | null } = { transactionHash: null, result: null }
+type Response = typeof response;
+
+class VerifiedContract {
+
+    constructor() {
+
+    }
+
+    private isNumber(value:number):boolean{
+        return !isNaN(value) && typeof value === 'number';
+      }
+    protected validateInput(type: DATATYPES, data: any) {
```

Comment:
@herberthk 

This validateInput function is to validate parameters sent by the SDK's clients/users to functions in contracts in /src/contract.

Reply:
@kallolborah 'GasLessTransaction' is also a contract despite the name, we shall discuss the naming latter. According to the design of this SDK  src/contract/index.ts is the base for all contracts however `class VerifiedContract ` has required parameters which are not applicable to `GasLessTransaction`. So I ended up creating another version of `class VerifiedContract ` in `src/contract/userop/index.ts` which has validation and other required methods if you check well you will see it. Tell me what you think about this?

Reply:
@herberthk 

yes, creating a different version of verified contract for userops is fine